src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/TelemetryUtility.cs

```diff
@@ -22,6 +23,9 @@ namespace NuGet.VisualStudio.Telemetry
 {
     public static class TelemetryUtility
     {
+        private static int FaultEventCount = 0;
+        public static int TotalFaultEvents => FaultEventCount;
```

Comment:
Just to make sure we don't overflow, I would use `ulong`:

```suggestion
        private static ulong FaultEventCount = 0;
        public static ulong TotalFaultEvents => FaultEventCount;
```

Reply:
I hardly doubt if we ever reach 4 billion fault events happen during VS instance run, I could be wrong.

Reply:
It does seem like overkill, but whatever. However, `Interlocked.Increment` only has overloads for `int` and `long`, none of the unsigned types.